app/grandchallenge/products/views.py

```diff
@@ -118,6 +119,7 @@ def get_queryset(self):
                 Q(),
             )
             queryset = queryset.filter(q)
+        self.queryset = queryset
```

Comment:
This should not be necessary

Reply:
If I currently search in the company bar for e.g. 'Thirona', one company comes up, but it states it found 50 results instead of 1 --> "company_total": len(self.queryset). Defining self.queryset solves this.

Reply:
Ah! Maybe try this instead: in `CompanyList.get_context_data` change `"company_total": len(self.queryset),` to `"company_total": len(context["object_list"]),`.

Reply:
fixed (also for ProductList)